provider/repodata/data/json/time_zone/exemplar_cities@1/ar-EG.json

```diff
@@ -90,6 +90,7 @@
   "cahal": "هاليفاكس",
   "caiql": "اكويلت",
   "camon": "وينكتون",
+  "camtr": "Montreal",
```

Comment:
Observation: Montreal is being added without translations in most locales

Reply:
Should I file a CLDR bug?

Reply:
Also I think it's better than not adding it, even in Arabic `Montreal` is probably more helpful than `camtr` or whatever else this falls back to.

Reply:
Filed https://unicode-org.atlassian.net/browse/CLDR-16489